### ISSUE TITLE
Is this repo deprecated?

### ISSUE THREAD

**Issue by linuxtim** [stm32-rs/stm32-usbd-examples#13]
The README for https://github.com/mvirkkunen/usb-device points to https://github.com/stm32-rs/stm32-usbd - which in turn points here for examples, but these examples are outdated vs. the examples in the individual hal crates e.g. https://github.com/stm32-rs/stm32f1xx-hal/blob/master/examples/usb_serial_rtic.rs is newer than https://github.com/stm32-rs/stm32-usbd-examples/blob/master/example-stm32f103c8/examples/serial_rtfm.rs

Should the README for this crate and stm32-usbd be updated to reflect this?

**Comment by Disasm** [comment]
Yes, this crate is outdated, most of the examples are already moved to HALs. It would be nice to update the README of stm32-usbd.
